conda/plan.py

```diff
@@ -23,17 +23,13 @@
 from .base.context import context, reset_context
 from .common.io import dashlist, env_vars, time_recorder
 from .common.iterators import groupby_to_dict as groupby
-from .core.index import LAST_CHANNEL_URLS, _supplement_index_with_prefix
-from .core.link import PrefixSetup, UnlinkLinkTransaction
-from .core.solve import diff_for_unlink_link_precs
-from .exceptions import CondaIndexError, PackagesNotFoundError
-from .history import History
+from .core.index import LAST_CHANNEL_URLS
+from .core.link import _get_best_prec_match, revert_actions  # noqa: F401
+from .deprecations import deprecated
 from .instructions import FETCH, LINK, SYMLINK_CONDA, UNLINK
 from .models.channel import Channel, prioritize_channels
 from .models.dist import Dist
 from .models.enums import LinkType
-from .models.match_spec import ChannelMatch
-from .models.prefix_graph import PrefixGraph
```

Comment:
Should we add `deprecated.constant(...)`s for all re-exported symbols this removes?